src/functions/components/FunctionRunsList.tsx

```diff
@@ -0,0 +1,45 @@
+// Libraries
+import React, {FC, useContext, useEffect} from 'react'
+import {useParams} from 'react-router-dom'
+
+// Components
+import {ComponentSize, EmptyState, IndexList} from '@influxdata/clockface'
+import {FunctionListContext} from 'src/functions/context/function.list'
+import FunctionRunsRow from './FunctionRunsRow'
+
+const FunctionRunsList: FC = () => {
+  const {id: functionID} = useParams<{id: string}>()
+  const {runsList, getRuns} = useContext(FunctionListContext)
+
+  useEffect(() => {
+    getRuns(functionID)
```

Comment:
should this happen in the provider when you set the current draft state?

Reply:
:( the draft state function and function to get runs for are different unfortunately. the routing branches off at functions list and you can go to edit a function or you can view its runs.